src/common/ChipCell/ChipCellView.js

```diff
@@ -27,73 +30,92 @@ const ChipCellView = React.forwardRef(
       show
     },
     ref
-  ) =>
-    (isEditMode || !isEveryObjectValueEmpty(chips)) && (
-      <div className="chips-wrapper" ref={ref}>
-        {chips.visibleChips.map((chip, index) => {
-          const { chipLabel, chipValue } = getChipLabelAndValue(chip)
+  ) => {
+    const buttonAddClassNames = classnames(
+      'button-add',
+      className && className,
```

Comment:
This is redundant.
You can do:
```suggestion
      className,
```